platforms/hyperledger-besu/charts/besu-node/templates/besu-config-configmap.yaml

```diff
@@ -27,7 +27,7 @@ data:
     node-private-key-file={{.Values.node.besu.privateKeyPath | quote }}
 
     # Transaction Pool
-    tx-pool-layer-max-capacity={{ .Values.node.besu.txPool.maxCapacity }}
+    tx-pool-max-size={{ .Values.node.besu.txPool.maxCapacity }}
```

Comment:
why are we changing this ?

Reply:
Due to the changes of the quorum version. They have renamed that variable

Reply:
Due to the changes of the quorum version. They have renamed that variable